src/charts/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js

```diff
@@ -166,6 +167,10 @@ export const SimplePowerBIReportEmbed = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [lang, scenario, reports.status]);
 
+  useEffect(() => {
+    report?.applyTheme({ themeJson: theme });
```

Comment:
Shouldn't we add the same condition `if (theme)` as below?

Reply:
I've added it, just in case, thanks.